books.js

```diff
@@ -1,61 +1,85 @@
 const titleInput = document.getElementById('title-input');
 const authorInput = document.getElementById('author-input');
 const submitButton = document.getElementById('submit-button');
-let books = [];
 
-function removeBook(book) {
-  books = books.filter((element) => JSON.stringify(element) !== JSON.stringify(book));
-  localStorage.setItem('books', JSON.stringify(books));
-}
+class Methods {
+  constructor() {
+    this.books = [];
 
-function removeBtnEventListener(element, title, author) {
-  const book = { title, author };
-  removeBook(book);
-  element.parentElement.remove();
-}
+    this.init = function () {
+      if (JSON.parse(localStorage.getItem('books')) !== null) {
+        this.books = JSON.parse(localStorage.getItem('books'));
+        this.books.forEach((element) => {
+          this.addBookToDocument(element);
+        });
+      }
+    };
 
-function addBookToDocument(book) {
-  const booksDiv = document.getElementById('books');
-  const bookDiv = document.createElement('div');
-  const title = document.createElement('p');
-  title.classList.add('title');
-  const author = document.createElement('p');
-  author.classList.add('author');
-  const remove = document.createElement('button');
-  remove.classList.add('remove-btn');
-
-  title.innerText = book.title;
-  author.innerText = book.author;
-  remove.innerText = 'Remove';
-
-  bookDiv.append(title, author, remove);
-  booksDiv.append(bookDiv);
-  remove.addEventListener('click', () => {
-    removeBtnEventListener(remove, book.title, book.author);
-  });
-}
+    this.removeBook = function (time) {
+      this.books = this.books.filter((book) => book.time !== time);
+      localStorage.setItem('books', JSON.stringify(this.books));
+    };
+
+    this.removeBtnEventListener = function (element, time) {
+      this.removeBook(time);
+      element.parentElement.remove();
+    };
+
+    this.addBookToDocument = function (book) {
+      const booksDiv = document.getElementById('books');
+
```

Comment:
- [ ] You have done a great job. It would be better if we define the class methods outside the constructor.
- When class methods are declared on the prototype (e.g. outside of the constructor) only one instance of that function exists and is re-used for all class instances, such that Methods.prototype.addBookToDocument === book1.addBookToDocument === book2.addBookToDocument. When methods are declared on the instance (e.g. inside of the constructor) a new function is created for every instance, such that book1.addBookToDocument!== book2.addBookToDocument
 I would recommend the following structure:
```js
class Books {
  constructor() {
    this.books = [];
  }

  addBook(title, author) {
  }

  removeBook(bookID) {
  }

  // getter method
  getBooks() {
  }
}
```